regulations/static/regulations/css/less/module/full-footer.less

```diff
@@ -0,0 +1,2 @@
+.reg-footer {
```

Comment:
I didn't have a really good idea of what the default attributes of a footer in an extendable them - so this is more or less just a skeleton file.  I only included this line to enforce a naming convention for footer css.  I added a few comments - let me know this suffices.